reference/library/src/morpheus-master/sass/modules/tutorial/_base.scss

```diff
@@ -0,0 +1,85 @@
+$alt-colour: #d36f00;
```

Comment:
I agree with Juanjo, this value should be on `_defaults.scss` on `sass` folder.

Everything else LGTM